packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  getFocusStyle,
+  HighContrastSelector
+} from '../../Styling';
+import {
+  ILinkStyleProps,
+  ILinkStyles
+} from './Link.types';
+
+export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
+  const { isButton, isDisabled, theme } = props;
+  const { semanticColors } = theme;
+
+  return {
+    root: [
+      'ms-Link',
+      getFocusStyle(theme),
+      {
+        color: semanticColors.link,
+      },
+      isButton && {
```

Comment:
it seems that className isn't mixed into root.

Reply:
This was intentional, as this component never had a `className` prop added and my understanding is that we're deprecating those eventually.

Reply:
My mistake, while `className` isn't explicitly listed in the props as it is for most components, it is expected to be there as the props extend from other HTML elements. I'll add this functionality in.